### DISCUSSION TITLE
set number inside the marker

### DISCUSSION THREAD

**Discussion by huseynehedov** [mapsforge/vtm#963] [Q&A] [answered]
I have many locations, and I create marker each of them, but I want to use sort by numbers, for example I have 3 locations and I put same marker but different numbers, marker above 1, marker above 2, marker above 3. How do I this? Please someone helps me, Thanks advanced.

**Comment by devemux86** [comment, marked as answer]
You can set different symbol per marker.

See the [MarkerLayerLabelsTest](https://github.com/mapsforge/vtm/blob/master/vtm-playground/src/org/oscim/test/MarkerLayerLabelsTest.java) and [AtlasMultiTextureTest](https://github.com/mapsforge/vtm/blob/master/vtm-playground/src/org/oscim/test/AtlasMultiTextureTest.java) samples.

**Reply by huseynehedov** [reply]
Thanks for all
